sensor/kubernetes/fake/fake.go

```diff
@@ -85,14 +85,33 @@ type WorkloadManager struct {
 	networkManager      manager.Manager
 }
 
+// WorkloadManagerConfig WorkloadManager's configuration
+type WorkloadManagerConfig struct {
+	workloadFile string
+	client       client.Interface
```

Comment:
Why do we have client here when no methods use that field? This suggests that maybe it should be a var outside of this struct (or shall simply be reamoved)?

Reply:
Oops that's left over from when I was spiking on this 😅 I'll remove it.